system-addon/content-src/styles/_theme.scss

```diff
@@ -0,0 +1,80 @@
+// Default theme
+.outer-wrapper {
```

Comment:
We can set `:not(.dark-theme)` to avoid having both styles apply with the latter overriding for dark theme. Not sure if there's a good way to make sure both lists define the same variables though…

Reply:
is there a performance gain to doing that? or just a little less clutter in dev tools inspector?

Reply:
John-Galt seems to suggest it probably won't make much of a difference performance-wise. So for developer / QA checking, I think we can stick **without** `:not` as the default theme `var`s will always be set, so if we accidentally forget to set it in `.dark`, it should be more obvious when a light theme's color shows up incorrectly.

Reply:
A problem with having the variables under `.outer-wrapper` is that snippets/onboarding won't be able to see them because they live outside of this in the DOM tree. So, we probably wan't to assign them to `:root` at that point, but then we will need a new way to override them for each theme because we can't set a `.dark-theme` class to `:root` can we? I guess we can cross that bridge when we get there. Maybe https://bugzilla.mozilla.org/show_bug.cgi?id=1444459 will be fixed soonish and we can just wait for that.